muacryptcc/plugin.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
```

Comment:
why :grey_question:

Reply:
just a general py2/py3 compat boilerplate.  there should also be unicode_literals but that breaks some of the tests (not sure why yet).

Reply:
@hpk42 but there is no print call in this file.